addins/cfsample2/README.md

```diff
@@ -2,4 +2,16 @@
 See https://officedev.github.io/custom-functions/LICENSE
 
 # Purpose
-This add-in is for internal testing purposes.
+This add-in is for internal testing purposes.It is used to test SharedRuntime with CustomFunction, AppCommand and Taskpane.
+This add-in contains:
+- Custom Functions
+- A ShowTaskpane button
+- A UI-less ribbon button handler
+
+This Addin contains three main Functionality:
+1. Taskpane: Contains six buttons, [Set/Read]Data- To set or get the value from the Shared global variable g_sharedAppData',  Btn[Get/Set]RtState- To get the Current Runtime state or set Current Runtime State to Visible or             Background and Btn[Get/Set]StartupState- To get Current Runtime startup state or set Current Runtime Startup state to Visible or background.
```

Comment:
1 space before and 1 space after the each hyphens.
Both 'To' (after the hyphens) should be 'to'.
The last 'background' should be 'Background' since that is a term.